.gitignore

```diff
@@ -14,5 +14,9 @@
 
 # ignore app key which contains email user name
 /config/initializers/app_keys.rb
+/config/initializers/google_cse_api.rb
```

Comment:
Initializers file that don't contain sensitive information shouldn't be in `.gitignore` only files that contain keys